{"uuid": "91b42324-d485-412b-a88b-0b5f3735a4c9", "children": ["ba557634-e59d-45bf-93b0-f2014cfdaaa6"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "RuntimeError: All attempts to create Remote driver failed. Last error: TypeError: WebDriver.__init__() got an unexpected keyword argument 'desired_capabilities'\n\n", "trace": "  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ~~~~~~~~~~~~~~~~~^^\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\pluggy\\_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\projects\\mobile-demo\\conftest.py\", line 97, in driver\n    drv = _create_ios_driver(\n        device_name=os.getenv('IOS_DEVICE_NAME', 'iPhone 15 Pro'),\n        platform_version=os.getenv('IOS_PLATFORM_VERSION', '17.0'),\n        app_id=os.getenv('IOS_APP_ID')\n    )\n  File \"C:\\projects\\mobile-demo\\conftest.py\", line 72, in _create_ios_driver\n    driver, method = _try_create_remote('https://hub.browserstack.com/wd/hub', caps)\n                     ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\projects\\mobile-demo\\conftest.py\", line 59, in _try_create_remote\n    raise RuntimeError('All attempts to create Remote driver failed. Last error: {}'.format(tb))\n"}, "start": 1757912398769, "stop": 1757912398769}], "start": 1757912398769, "stop": 1757912398908}